packages/nodes-base/nodes/Anclap/actions/transactions/withdraw/description.ts

```diff
@@ -136,13 +133,12 @@ export const withdrawDescription: INodeProperties[] = [
 		},
 		default: '',
 		description:
-			'Type of memo. One of text, id or hash. Deprecated because memos used to identify users of the same Stellar account should always be of type of id.',
+			'Type of memo. One of text, ID or hash. Deprecated because memos used to identify users of the same Stellar account should always be of type of ID.',
```

Comment:
```suggestion
			'Type of memo. One of text, ID or hash. Deprecated because memos used to identify users of the same Stellar account should always be of type of ID',
```